llvm/include/llvm/Transforms/Utils/CodeExtractor.h

```diff
@@ -133,6 +137,25 @@ class CodeExtractorAnalysisCache {
     // space.
     bool ArgsInZeroAddressSpace;
 
+    // If set, this callback will be used to allocate the arguments in the
+    // caller before passing it to the outlined function holding the extracted
+    // piece of code.
```

Comment:
```suggestion
    /// If set, this callback will be used to allocate the arguments in the
    /// caller before passing it to the outlined function holding the extracted
    /// piece of code.
```
These can be doxygen comments